jupyterhub/schema.yaml

```diff
@@ -1,55 +1,21 @@
 title: Config
 type: object
 properties:
-  hub:
+  global:
```

Comment:
`schema.yaml` in its entirety is mostly just documentation, but its also some adjustment to avoid repeating the same stuff over and over by using YAML anchors (`$`) and references (`*`) to the previously defined anchors.

This section could be reviewed in the PR review by RTD. See https://zero-to-jupyterhub--1794.org.readthedocs.build/en/1794/reference/reference.html#global